semiot/ontologies/hydra/.htaccess

```diff
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^hydra-pubsub$ https://raw.githubusercontent.com/semiotproject/ontologies/master/hydra-pubsub/hydra-pubsub.ttl [R=303,L]
+RewriteRule ^hydra-filter$ https://raw.githubusercontent.com/semiotproject/ontologies/master/hydra-filter/hydra-filter.ttl [R=303,L]
```

Comment:
These will redirect https://w3id.org/semiot/ontologies/hydra/hydra-xxx, did you mean for the rule to just be `^xxx$` like the README implies?